packages/flutter_test/lib/src/controller.dart

```diff
@@ -423,7 +423,11 @@ abstract class WidgetController {
   /// See [PointerEventRecord].
   Future<List<Duration>> handlePointerEventRecord(List<PointerEventRecord> records);
 
-  /// Called to indicate that time should advance.
+  /// Called to indicate that there should be a new frame after an optional
+  /// delay.
```

Comment:
we should probably make it clear that the time that is advanced is the FakeAsync clock when you're using this in `flutter test`, and the real clock in live environments.

Reply:
Similar things are already in the xxxTestBinding.pump. I added a sentence in `WidgetTester` for this but I don't think it should be in `WidgetController` because `WidgetController` does not necessarily come with a customized binding and there isn't the context about FakeAsync because controller is assuming the binding to be general.